src/CommonITILActor.php

```diff
@@ -48,11 +48,17 @@ abstract class CommonITILActor extends CommonDBRelation
     public const ASSIGN    = 2;
     public const OBSERVER  = 3;
 
+    /**
+     * @return string
```

Comment:
`$items_id_1` and `$items_id_2` are marked as nullable in parent class; so should be nullable here too